lib/actions.js

```diff
@@ -1616,16 +1633,34 @@ exports.waitFor = function() {
 /**
  * Waits for a function to evaluate to a given value on the given page
  * @param {Page | undefined} page
- * @param {function} fn
+ * @param {function | object} optsOrFn - If optsOrFn is a function, use the classic signature waitForPage(page, fn, arg1, arg2, value), If arg is an object, use waitForPage(page, options)
+ * @param {array} optsOrFn.args - arguments of fn
+ * @param {function} optsOrFn.fn - function to evaluate
+ * @param {function} optsOrFn.fname - function name (can be set with fn.name on the classic signature)
```

Comment:
I have my neurotic issues also and I do not like the synthax

```
key : function nameOfTheFunction(){
}
```
And i prefer
```
key : function nameOfTheFunction(){
}
```

That's why :-)

Anyway, i removed it in 93a2ad6